packages/govuk-frontend/src/govuk/components/pagination/pagination.yaml

```
@@ -36,7 +36,11 @@ params:
       - name: text
         type: string
         required: false
-        description: The link text to the previous page. Defaults to 'Previous'.
+        description: The text content of the link to the previous page. If `previous.html` is supplied, this is ignored.
```

We seem to add the "Defaults to" text in both `html` and `text`?

```suggestion
        description: The text content of the link to the previous page. Defaults to 'Previous page', with 'page' being visually hidden. If `previous.html` is supplied, this is ignored.
```

---

I'm not sure that we have a convention, personally. We have very few instances where we provide default text **and** have both a `text` and a `html` option. From what I can see skimming through:

* Back Link documents the default (which is plain text) in both `text` and `html` parameters.
* Notification Banner documents the plain text default(s) in `titleText`, omitting them from `titleHtml`. 
* Footer documents the (HTML-containing) defaults in both text and HTML parameters of `contentLicence` and `copyright`.

Personally I think the documentation on what is default should be stuck with whatever parameter is most relevant—`text` if it's plain text, `html` if it contains HTML—especially in cases where it can be quite long, like on the Notification Banner, but it's not something I have strong feelings over.

---

I'd stick it in both, knowing that this text might go into editor/IDE autocomplete in future

But yep, we definitely have gaps where we should _also_ put it in both

Probably better to repeat ourselves that assume it's obvious from the context?